### ISSUE TITLE
GoFundMe to restart development on this?

### ISSUE THREAD

**Issue by billynoah** [sequelpro/sequelpro#3684]
I use this software every day and have donated in the past.  I think there are a lot of people like me who love Sequel Pro and would happily donate again if we knew that the dev team existed and was committed to the project.  Can any of the long time contributors put a dollar value on what it would take to get things going again?  $100k?  $200k?  It might be worth a shot to try and raise money.

**Comment by shanecp** [comment]
Who's in the core team for this? Are they still active?

**Comment by Hubertnator** [comment]
@dmoagx, @stuconnolly  and other people. @dmoagx wasn't active for the last two months, @stuconnolly's last commit was 3 days ago.

I don't think money is a problem, but it is worth the try.

**Comment by rowanbeentje** [comment]
The main contributor for years now has been @dmoagx .

However, the project is currently still "owned" by @abhibeckert and @mattlangtree , and they run the website and receive donations. Any fundraising effort should probably focus first on a move to community ownership, or stewardship.

**Comment by shanecp** [comment]
I guess the current donations for this is not enough to motivate to dedicate more time. But I think a one-off campaign may not be enough to sustain it for over 2+ years. Perhaps moving to a community + paid for teams might be a sustainable option.

**Comment by dmoagx** [comment]
In the past I've estimated the cost for a PostgreSQL implementation by a freelance developer to be around 60k€, which back then was based on the assumption that it would be around 1 year worth of work for a mid range developer in Germany.

The total amount of donations between ca. 2009 to 2014 or so (back when the project was much more active) was around 10k€ IIRC.

Just to give you a perspective on how realistic it would be to raise the required amount of money...

**Comment by billynoah** [comment]
That's good information @dmoagx - I suspect that people may have been a bit lazy about donating to an existing & active project when they don't really have to, but faced with the reality of something that will cease to exist without support may draw a different attitude.  Who knows, I could be wrong.  I can only speak for myself personally and I know that I would double or triple my investment at this stage because this application is important to me and it's plain to see that development has fallen by the wayside.

**Comment by pedzed** [comment]
I'm considering creating an open-source, cross-platform, cross-database alternative with Go and [Wails](https://wails.app).

Where can I read more about people's requirements?

**Comment by shanecp** [comment]
@dmoagx I thought that would be the case. Donations won't be enough to sustain the development. 

You've mentioned it was much more active in 2014. Do you know what changed since then? Is it because there are more similar products, or are developers moving to more commercial options? or are people moving away from MySQL?

**Comment by mattlangtree** [comment]
> The main contributor for years now has been @dmoagx .
> 
> However, the project is currently still "owned" by @abhibeckert and @mattlangtree , and they run the website and receive donations. Any fundraising effort should probably focus first on a move to community ownership, or stewardship.

I'm very happy to assist with transferring ownership to the community or any suitably interested group/individual. I'm not able to dedicate much time, but I can help unblock or handover if interested parties want to take up the baton. I still do have access to a lot of accounts and I can ask Abhi to assist I'm sure. 

To build on what Rowan said, I think we need someone with leadership or a real determined party to take this on. You've got to be in it for more than just the code, which is what a lot of us are here for.

**Comment by fuzzy76** [comment]
I'd be happy to donate to a living project. But as the project is now, I would consider it a waste of money. They would be better spent on a commercial alternative.

You need a working team before taking donations, it can't be done the other way around.

**Comment by billynoah** [comment]
@fuzzy76 - I agree.  The idea here is that financial support could possibly motivate the project back to life.  As a freelance developer myself, it's easy to imagine how difficult it would be to donate time and energy to this project while there are paying, full time jobs about.

**Comment by brandon-bt** [comment]
@dmoagx Just a note on some of the psychology and motivations behind donations - I have donated to the project personally, but the amount I am willing to donate as a "thank you" to a project that appeared to no longer receiving updates vs. the amount I would be willing to donate to a team that was **actively** committed to sustaining the project for a while longer differs by a substantial amount.

I don't know what the most effective community support model looks like. What I do know is there appears to be broad support for one.

Sequel Pro is excellent. It's nimble. It "gets out of the way" and lets me deal with database environments like no other app I have tried (and I've tried many). I'm desperate to keep it as long as I can. If there is any effort to gather funds to move development forward, I will contribute immediately.

**Comment by shazron** [comment]
> Any fundraising effort should probably focus first on a move to community ownership, or stewardship.
I agree. Anything less we will end up with another thread similar to this one in the future. There is already a fork: https://github.com/Sequel-Ace/Sequel-Ace but it needs a governance model as well if not it faces the same fate. 

Some guidance: https://opensource.guide/leadership-and-governance/#what-are-some-of-the-common-governance-structures-for-open-source-projects

**Comment by JPeterson2015** [comment]
Restart this thing!!! I use it all the time and it needs to be updated!

**Comment by Kaspik** [comment]
Please read about the future in new topic - https://github.com/sequelpro/sequelpro/issues/3705

**Comment by abhibeckert** [comment]
Sequel Pro hasn't been worked on in a long time, but the project isn't dead and I do intend to continue development.

As for what it would cost, it's not a matter of money (though obviously money is helpful). It's a matter of people with free time to contributing to the project. I personally haven't had nearly as much time as I would like, but I have always put as much time as I can into Sequel Pro and will continue to do so. For example since WWDC I've been working out exactly what is required to be done to ensure the app works well on both Mac OS 11 and on ARM processors.

I'm also working on things with my career and family life and expect to be able to dedicate a lot more time to Sequel Pro in the near future.

**Comment by fuzzy76** [comment]
Sequel Pro haven't seen a release for four years. Meanwhile, Sequel Ace (the fork) has seen 85 releases and multiple contributors this year alone.

There has never been a shortage of people with free time to contribute with code to Sequel Pro. It just doesn't work when pull requests are ignored and nothing is released. For any moderate sized project to survive, you can't rely on a single point of failure.
